app.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+
+function setLocalStorage(bookList) {
+  const updateBooks = JSON.stringify(bookList);
+  localStorage.setItem('#book-form', updateBooks);
+}
+
+function getLocalStorage() {
+  const existingBooks = localStorage.getItem('#book-form');
+  if (existingBooks !== null) {
+    return JSON.parse(existingBooks);
+  }
+  return [];
+}
+const booksList = getLocalStorage();
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const container = document.querySelector('.book-list');
+
+function displaybooks(books) {
+  container.innerHTML = '';
+  for (let i = 0; i < books.length; i += 1) {
+    const book = books[i];
+    const maindiv = document.createElement('div');
+    const h2 = document.createElement('p');
+    const p = document.createElement('p');
+    const removeButton = document.createElement('button');
+    removeButton.addEventListener('click', () => {
+      booksList.splice(i, 1);
+      setLocalStorage(booksList);
+      displaybooks(booksList);
```

Comment:
Great job team 🤝

- [ ] [Optional] As stated in the project requirements, kindly create functions to both add and delete books. You may make these methods independent and implement them separately.